lib/rocket_pants.rb

```diff
@@ -84,9 +84,12 @@ def show_exception_message=(value)
     end
 
     def default_env
-      env = Rails.env.to_s if defined?(Rails.env)
-      env ||= ENV['RAILS_ENV'].presence || ENV['RACK_ENV'].presence || "development"
-      ActiveSupport::StringInquirer.new env
+      if defined?(Rails.env) && Rails&.env
```

Comment:
You shouldn't have to use `&.` since you just used `Rails`...

Reply:
yeah I can remove it, np